geode-core/src/integrationTest/resources/org/apache/geode/codeAnalysis/sanctionedDataSerializables.txt

```diff
@@ -351,7 +351,7 @@ fromData,16
 toData,16
 
 org/apache/geode/distributed/internal/streaming/StreamingOperation$StreamingReplyMessage,2
-fromData,414
+fromData,420
```

Comment:
This will not affect the serialization contract correct?  The number of bytes only changes because the TestHook changed in the fromData() method, but the actual serialization logic didn't change.  Tagging a couple people who know more about serialization than I do for confirmation we aren't introducing a backwards compatibility issue here: @galen-pivotal @upthewaterspout

Reply:
As long as the behavior is the same we're fine. This code analysis is really just a heuristic to try to prevent backwards compatibility changes.